listenbrainz/db/stats.py

```diff
@@ -52,11 +53,10 @@ def insert_user_stats(user_id, artists, recordings, releases, artist_count):
              artist_count (int): the total number of artists listened to by the user
     """
 
+    artist_range = artists['range']
+    del artists['range']
     artist_stats = {
-        'count': artist_count,
-        'all_time': {
-            'artists': artists,
-        }
+        artist_range: artists
```

Comment:
this json should also contain the timestamp for which the stats are. We should be able to easily find out and show that the stats are for the month of July or for the week of May 18.